lib/game.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '..\lib\player.rb'
+require_relative '..\lib\board.rb'
+class Game
+  attr_reader :player1, :player2, :board
```

Comment:
also place :turn and :result_combinations here as you are needing it in other methods below